app/javascript/react/components/Map/Markers/MobileMarkers.tsx

```diff
@@ -115,14 +114,13 @@ const MobileMarkers = ({
     if (isOverlapping) {
       // Display as a dot when markers are too close
       return (
-        <MarkerContainer
+        <DotMarker
+          color={red}
           onClick={() => {
             onMarkerClick(Number(session.point.streamId), Number(session.id));
             centerMapOnMarker(session.point);
           }}
-        >
-          <StreamMarker color={red} />
-        </MarkerContainer>
```

Comment:
I separated the stream marker from the dot marker visible when multiple markers are overlapping. Styles are mostly copied from SingleMarker to ensure the dot is visible in the same position.